core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -3257,6 +3257,14 @@ private[spark] object Utils extends Logging {
       case _ => math.max(sortedSize(len / 2), 1)
     }
   }
+
+  def closeAll(closeables: AutoCloseable*): Unit = {
```

Comment:
I think this is too much to have it as a common util at the core module. It's only used twice ..